### PR TITLE
Add amp-pixel sample

### LINKED ISSUE #253: Sample for amp-pixel

### PR DESCRIPTION
**Pull request by jmadler**
Resolves #253

### CODE REVIEW

File: src/20_Components/amp-pixel.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!--
+  ## Introduction
+
+  amp-pixel is a lightweight mechanism for measuring pageviews.  It provides
```
Comment (sebastianbenz) [inline src/20_Components/amp-pixel.html]: Can you link `amp-pixel` to the doc page.

File: src/20_Components/amp-pixel.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  functionality similar to amp-analytics, but only includes a single event
```
Comment (sebastianbenz) [inline src/20_Components/amp-pixel.html]: Can you link amp-analytics to the amp-analytics docs.

File: src/20_Components/amp-pixel.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<body>
+
+  <!-- ## Basic Usage -->
+  <!--
+    The amp-pixel issues a simple GET request to the given URL and ignores the result.  This is a fast way to track pageviews.
+  -->
+  <amp-pixel src="https://ampbyexample.com/tracker/foo" layout="nodisplay"></amp-pixel>
+
+  <!--
+    The variables noted in <a href="https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md">amp-var-substitutions</a> will be interpolated in the URL as a way to capture some meaningful data.
```
Comment (sebastianbenz) [inline src/20_Components/amp-pixel.html]: You can also use markdown syntax for links.

File: src/20_Components/amp-pixel.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  -->
+  <amp-pixel src="https://ampbyexample.com/tracker/foo?redirectTime=REDIRECT_TIME&sourceHost=SOURCE_HOSTNAME&viewer=VIEWER" layout="nodisplay"></amp-pixel>
+
+  <!--
+    The optional "referrerpolicy" attribute allows you to strip the 'Referrer' header in the outbound HTTP request.  Other values of referrerpolicy that are valid for other tags (e.g. amp-img) are not available for amp-pixel.
```
Comment (sebastianbenz) [inline src/20_Components/amp-pixel.html]: This seems to cause a validation error.

Comment (jmadler) [inline src/20_Components/amp-pixel.html]: Yep, noted in https://github.com/ampproject/amphtml/issues/8979

**Pull request merged**